src/controllers/user-controller.js

```diff
@@ -120,10 +126,33 @@ const deleteUser = async (req, res) => {
   }
 };
 
+const setNewPassword = async (req, res) => {
+  const result = newPass.validate(req.body);
+  if (result.error) {
+    res.status(400).send({
+      status: 'FAILED',
+      data: { error: result.error.details },
+    });
+  } else {
+    try {
+      await userService.setNewPassword(result.value);
+      res
+        .status(202)
```

Comment:
why 202?

Reply:
im goin to change the code for 200, i was thinking that the code 202 was the correct one